plugin/symbols.py

```diff
@@ -109,13 +108,22 @@ def handle_response(self, response: Union[List[DocumentSymbol], List[SymbolInfor
         self.view.settings().erase(SUPPRESS_INPUT_SETTING_KEY)
         window = self.view.window()
         if window and isinstance(response, list) and len(response) > 0:
+            panel_items = self.process_symbols(response)
```

Comment:
I hate code that creates side effects (self.regions) and also returns something but decided not to refactor anything here.